DEV-INTRO.md

```diff
@@ -10,19 +10,26 @@ To clone the PEcAn repository:
 
 ```sh
 git clone git@github.com:pecanproject/pecan
-
+cd pecan
 # alternatively, if you haven't set up ssh keys with GitHub
 # git clone https://github.com/PecanProject/pecan
 ```
 
 ## Developing in Docker
 
+### Installing Docker 
+
+To install Docker and docker-compose, see the docker documentation for installing 
```

Comment:
maybe refer to the docker documentation in the manual as well.

Reply:
done